tests/templates/kuttl/cert-manager-tls/consumer.yaml

```diff
@@ -18,7 +18,7 @@ spec:
               set -euo pipefail
               ls -la /stackable/tls
               cat /stackable/tls/tls.crt | openssl x509 -noout -text
-              cat /stackable/tls/tls.crt | openssl x509 -noout -text | grep "DNS:my-tls-service.$NAMESPACE.svc.cluster.local"
+              cat /stackable/tls/tls.crt | openssl x509 -noout -text | grep -P "DNS:my-tls-service.$NAMESPACE.svc.cluster.local\.?"
```

Comment:
This will just test that _either_ variant exists.

Reply:
Ah I though this is just a debug statement and not an assertion :see_no_evil: 
d4f4ad7 (and yes, the assert is not optimal but didn't find a nice way in a realistic time frame)